bidsify/bidsify.py

```diff
@@ -10,71 +10,104 @@
 import pandas as pd
 from heudiconv.main import workflow as heudiconv
 from bidsutils.metadata import complete_jsons, clean_metadata
+
 # Local imports
 from bidsify.utils import run, load_dicomdir_metadata, clean_tempdirs
 
 
 def _get_parser():
     """Set up argument parser for scripts."""
-    parser = argparse.ArgumentParser(description='BIDS conversion and '
-                                                 'anonymization.')
-    parser.add_argument('-d', '--dicomdir',
-                        type=Path,
-                        required=True,
-                        dest='dicomdir',
-                        help='Directory or tar file containing raw data.')
-    parser.add_argument('-f', '--heuristic',
-                        type=str,
-                        dest='heuristic',
-                        metavar='HEUR',
-                        help='Path to a heuristic file or name of a builtin '
-                             'heudiconv heuristic.')
-    parser.add_argument('-s', '--sub',
-                        required=True,
-                        dest='subject',
-                        help='The label of the subject to analyze.')
-    parser.add_argument('-ss', '--ses',
-                        required=False,
-                        dest='session',
-                        help='Session identifier',
-                        default=None)
-    parser.add_argument('-o', '--output_dir',
-                        type=Path,
-                        dest='output_dir',
-                        required=True,
-                        metavar='PATH',
-                        help="BIDS dataset directory. Must be either in "
-                             "scratch, the user's home directory, or within "
-                             "the current working directory.")
-    parser.add_argument('-w', '--work_dir',
-                        type=Path,
-                        dest='work_dir',
-                        required=False,
-                        metavar='PATH',
-                        default=None,
-                        help='Working directory (in scratch).')
-    parser.add_argument('--datalad',
-                        required=False,
-                        action='store_true',
-                        default=False,
-                        help='Use datalad to track changes to dataset.')
-    parser.add_argument('--username',
-                        required=False,
-                        metavar='NAME',
-                        nargs='+',
-                        default=None,
-                        help='GitHub username, for datalad.')
-    parser.add_argument('--useremail',
-                        required=False,
-                        metavar='EMAIL',
-                        default=None,
-                        help='GitHub email, for datalad.')
+    parser = argparse.ArgumentParser(
+        description="BIDS conversion and " "anonymization."
```

Comment:
```suggestion
        description="BIDS conversion and anonymization."
```

An annoying thing about black.

Reply:
I actually see a few instances. Perhaps try finding and replacing instances of `" "` with nothing.